remedy.lic

```diff
@@ -228,7 +228,6 @@ class Remedy
         get_item('sieve')
         command = "push my #{@noun} with my sieve"
       end
-    crush(command)
```

Comment:
It seems like this would now stop crafting after analyzing. Do you need to return `command` here and do something like `command = analyze_crush` above?